submit_cgap/submission.py

```diff
@@ -240,11 +241,63 @@ def get_defaulted_lab(lab, user_record, error_if_none=False):
     return lab
 
 
+def get_defaulted_consortia(consortia, user_record, error_if_none=False):
+    """
+    Returns the given consortia or else if none is specified, it tries to infer any consortia.
+
+    :param consortia: a list of @id's of consortia, or None
+    :param user_record: the user record for the authorized user
+    :param error_if_none: boolean true if failure to infer any consortia should raise an error, and false otherwise.
+    :return: the @id of a consortium to use (or a comma-separated list)
+    """
+    consortia = consortia
+    if not consortia:
+        consortia = [consortium.get('@id', None)
+                     for consortium in user_record.get('consortia', {})]
```

Comment:
I believe this should default to a list, though no functional difference. Similarly for line 281 for submission centers.
```suggestion
                     for consortium in user_record.get('consortia', [])]
```

Reply:
Yeah, since in either case it will do a null iteration, it hardly matters, but it will be more harmonious to use `[]`. :)